geoips/interfaces/base.py

```diff
@@ -461,7 +408,38 @@ def get_plugin(self, name):
         'name'.
         """
         try:
-            validated = self.validator.validate(self._unvalidated_plugins[name])
+            if isinstance(name, tuple):
+                plugin = yaml.safe_load(
+                    open(self._unvalidated_plugins[self.name][name[0]]["abspath"], "r")
+                )
+                for product in plugin["spec"]["products"]:
+                    if product["name"] == name[1]:
+                        plugin = product
+                        break
+                plugin["interface"] = "products"
+                plugin["abspath"] = self._unvalidated_plugins[self.name][name[0]][
+                    "abspath"
+                ]
+                plugin["relpath"] = self._unvalidated_plugins[self.name][name[0]][
+                    "relpath"
```

Comment:
@evrose54 did you get a full 0 return from tests/integration_tests/full_test.sh?  I think template_fusion_plugin and template_basic_plugin never worked with these updates?  I think some of the "old" code needs to be re-implemented in create_plugin_registry.

Reply:
See: https://github.com/NRLMMD-GEOIPS/geoips/pull/380/files
Resolve this conversation if you are good with those changes.